AdminServer/appscale/admin/__init__.py

```diff
@@ -0,0 +1,400 @@
+""" A server that handles application deployments. """
+
+import argparse
+import logging
+import socket
+import sys
+import time
+
+from appscale.common import appscale_info
+from appscale.common.constants import HTTPCodes
+from appscale.common.constants import LOG_FORMAT
+from appscale.common.ua_client import UAClient
+from appscale.common.ua_client import UAException
+from appscale.common.unpackaged import APPSCALE_PYTHON_APPSERVER
+from tornado import gen
+from tornado.options import options
+from tornado import web
+from tornado.escape import json_decode
+from tornado.escape import json_encode
+from tornado.ioloop import IOLoop
+from . import utils
+from . import constants
+from .constants import (
+  CustomHTTPError,
+  MAX_DEPLOY_TIME,
+  OperationTimeout,
+  REDEPLOY_WAIT,
+  VALID_RUNTIMES
+)
+from .operation import CreateVersionOperation
+from .operations_cache import OperationsCache
+
+sys.path.append(APPSCALE_PYTHON_APPSERVER)
+from google.appengine.api.appcontroller_client import AppControllerException
+
+
+# The state of each operation.
+operations = OperationsCache()
+
+
+@gen.coroutine
+def wait_for_port_assignment(operation_id, deadline, acc):
+  """ Waits until port is assigned for version.
```

Comment:
It's just a conceptual change. The idea now is that we will start by supporting just the default service/version. Once that conceptual change is there, it will be much easier to support multiple services and versions.